src/packet/checksum.rs

```diff
@@ -0,0 +1,21 @@
+//! rfc1071 checksum abstraction
```

Comment:
This file should have the same header as the other files. I also don't know that this is the right place for this code, perhaps `pnet::util` would be a better location.